akka-sample-cqrs-scala/src/main/scala/sample/cqrs/Main.scala

```diff
@@ -49,56 +60,100 @@ object Main {
        """).withFallback(ConfigFactory.load())
 
   /**
-   * To make the sample easier to run we kickstart a Cassandra instance to
-   * act as the journal. Cassandra is a great choice of backend for Akka Persistence but
-   * in a real application a pre-existing Cassandra cluster should be used.
-   */
+    * To make the sample easier to run we kickstart a Cassandra instance to
+    * act as the journal. Cassandra is a great choice of backend for Akka Persistence but
+    * in a real application a pre-existing Cassandra cluster should be used.
+    */
   def startCassandraDatabase(): Unit = {
     val databaseDirectory = new File("target/cassandra-db")
-    CassandraLauncher.start(databaseDirectory, CassandraLauncher.DefaultTestConfigResource, clean = false, port = 9042)
+    CassandraLauncher.start(
+      databaseDirectory,
+      CassandraLauncher.DefaultTestConfigResource,
+      clean = false,
+      port = 9042
+    )
   }
 
   def createTables(system: ActorSystem[_]): Unit = {
-    val session = CassandraSessionRegistry(system).sessionFor("alpakka.cassandra")
+    val session =
+      CassandraSessionRegistry(system).sessionFor("alpakka.cassandra")
 
     // TODO use real replication strategy in real application
-    val keyspaceStmt = """
+    val keyspaceStmt =
+      """
       CREATE KEYSPACE IF NOT EXISTS akka_cqrs_sample
       WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 }
       """
 
     val offsetTableStmt =
       """
-      CREATE TABLE IF NOT EXISTS akka_cqrs_sample.offsetStore (
-        eventProcessorId text,
-        tag text,
-        timeUuidOffset timeuuid,
-        PRIMARY KEY (eventProcessorId, tag)
+      CREATE TABLE IF NOT EXISTS akka_cqrs_sample.offset_store (
+        projection_name text,
+        partition int,
+        projection_key text,
+        offset text,
+        manifest text,
+        last_updated timestamp,
+        PRIMARY KEY ((projection_name, partition), projection_key)
```

Comment:
we will have a `createTable` utility on `CassandraProjection`, but this works for now